server/utils/utils.ts

```diff
@@ -0,0 +1,21 @@
+function properCase(word: string): string {
```

Comment:
As a stylistic preference I perfer to use function expressions with arrow functions to explicitly state the functions will not be renamed and it will not rely on an execution context.